README.md

```diff
@@ -20,15 +20,16 @@ Documentation
 
 ## Installation
 
-**1.**  Add dependency with composer
+**1.**  Add dependency with Composer
 
 ```bash
 composer require kunicmarko/sonata-auto-configure-bundle
 ```
 
-**2.** Register the bundle in your `bundles.php`
+**2.** Register the bundle in the kernel of your application:
```

Comment:
why would you say kernel here? since it should be added in the bundles file?

Reply:
ye, I know, but not sure most of the other people know that that is why I wrote `bundles.php`

Reply:
looks good